### PR TITLE
Fixes #1633 : Configured Codecov to the Project

### LINKED ISSUE #1633: No Code Coverage for the Project
**Actual Behaviour**
No Code Coverage for the Project

**Expected Behaviour**
Add Code Coverage for the Project

**Steps to reproduce it**
A new file named codecov.yml

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by naveentherocker**
Fixes #1633 
Configured Codecov to the Project

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -76,6 +76,7 @@ before_script:
 script:
   - ./scripts/copy_opencv.sh
   - ./scripts/main_script.sh
+  - ./gradlew build jacocoTestReport
```
Comment (abishekvashok) [inline .travis.yml]: Do we require this?

Comment (naveentherocker) [reply .travis.yml]: I think this Plugin is very useful for this Project

**Comment by naveentherocker** [comment]
@anantprsd5 Review Please.

File: .travis.yml (hunk 1 above)
Comment (harshit98) [reply .travis.yml]: @naveentherocker How it can be useful?

Comment (naveentherocker) [reply .travis.yml]: @harshit98 It Creates a code coverage report for tests of a single project in multiple formats.

Comment (abishekvashok) [reply .travis.yml]: @naveentherocker pls use this instead: `./gradlew connectedAndroidTest jacocoTestReport` and note that this should be done in `https://github.com/fossasia/phimpme-android/blob/development/scripts/main_script.sh`. I can see the stack trace is generated there as well. if you want you can add `jacocoTestReport`.

File: .travis.yml [hunk 2]
```diff
@@ -76,6 +76,7 @@ before_script:
 script:
   - ./scripts/copy_opencv.sh
   - ./scripts/main_script.sh
+  - ./gradlew build jacocoTestReport connectedAndroidTest
```
Comment (abishekvashok) [inline .travis.yml]: As I mentioned this should go into the `main-script.sh` script and not her on `.travis.yml`

File: .travis.yml (hunk 1 above)
Comment (abishekvashok) [reply .travis.yml]: The giff still shows the addition here. @naveentherocker

**Comment by naveentherocker** [comment]
@abishekvashok I think now it's fine

**Comment by abishekvashok** [comment]
@naveentherocker sorry for maiing your time a lot into this but the giff still shows an addition in .travis.yml; After you have resolved it, I will approve it. Note: It would be professional if u could squash as well but it's fine if you don't.

**Comment by naveentherocker** [comment]
@abishekvashok This is the only line which i have added `- ./gradlew build jacocoTestReport`

**Comment by abishekvashok** [comment]
You have modified main.sh and that's enough

**Comment by naveentherocker** [comment]
@abishekvashok Hope you approve it

File: .travis.yml (hunk 1 above)
Comment (abishekvashok) [reply .travis.yml]: @naveentherocker pls remove this change

File: scripts/main_script.sh [hunk 3]
```diff
@@ -6,7 +6,7 @@ set -e
 if [ "$COMPONENT" == "BUILD" ]; then
     ./gradlew build
 elif [ "$COMPONENT" == "CONNECTED_TEST" ]; then
-    ./gradlew build connectedAndroidTest --stacktrace
+    ./gradlew build connectedAndroidTest jacocoTestReport --stacktrace
```
Comment (harshit98) [inline scripts/main_script.sh]: Remove `jacocoTestReport` from here too.

Comment (naveentherocker) [reply scripts/main_script.sh]: @abishekvashok Wanted it to be there.

Comment (naveentherocker) [reply scripts/main_script.sh]: jacocoTestReports helps to collect unit test code coverages quickly

Comment (abishekvashok) [reply scripts/main_script.sh]: @harshit98 I guess he should put it here as test coverage should be reported by jacoco for codecov for smooth working.

Comment (harshit98) [reply scripts/main_script.sh]: ok great!

**Pull request merged**